src/core/lib/attribute-manager.js

```diff
@@ -620,4 +634,31 @@ export default class AttributeManager {
       }
     }
   }
+
+  /**
+   * Set transition options
+   * @params {Object} opts - transition options
+   * Returns updated attributes
+   */
+  setTransitionOptions(opts) {
+    if (!opts) {
+      this.attributeTransition = null;
+    } else if (this.attributeTransition) {
+      this.attributeTransition.setOptions(opts);
+    } else if (this.isTransitionSupported) {
+      this.attributeTransition = new AttributeTransitionManager(this, opts);
```

Comment:
Nit: can we change the name from `attributeTransition` to `transitionManger` or `attributeTranstionManger` for readability?